### DISCUSSION TITLE
router lib and nicegui

### DISCUSSION THREAD

**Discussion by bobanovo** [zauberzeug/nicegui#2971] [Q&A]
### Question

Hi, I know that this is simple question - but in some discussion here, I saw use "from router import Router" - exist any description why, how it use in nicegui app? Thanks for tips, links, etc...

**Comment by rodja** [comment]
It's from the [single page app example](https://github.com/zauberzeug/nicegui/tree/main/examples/single_page_app). Just keep in mind that we will shortly release built-in SPA support in NiceGUI (see #2811).
